crates/types/src/config/common.rs

```diff
@@ -441,6 +441,12 @@ pub struct CommonOptions {
     #[cfg_attr(feature = "schemars", schemars(skip))]
     #[serde(skip_serializing_if = "std::ops::Not::not", default)]
     pub experimental_enable_vqueues: bool,
+
+    /// # Ingestion Options
+    ///
+    /// Settings for the ingestion client, which is shared across components
+    /// such as Kafka ingress and the admin API.
+    pub ingestion_options: IngestionOptions,
```

Comment:
Should this maybe go to `IngressOptions`? Also wondering whether we can find a simpler name (e.g. just `ingestion`)?